pkg/pilot/cassandra/v3/pilot.go

```diff
@@ -39,9 +45,65 @@ func NewPilot(opts *PilotOptions) (*Pilot, error) {
 		nodeTool:            opts.nodeTool,
 	}
 
+	// hack to test the seedprovider works in k8s, this should be moved somewhere better
+	cfgPath := "/etc/cassandra/cassandra.yaml"
+	read, err := ioutil.ReadFile(cfgPath)
+	if err != nil {
+		return nil, err
+	}
+
+	newContents := strings.Replace(string(read),
+		"org.apache.cassandra.locator.SimpleSeedProvider",
+		"io.k8s.cassandra.KubernetesSeedProvider", -1)
+
+	newContents = strings.Replace(string(newContents),
+		"SimpleSnitch",
+		"GossipingPropertyFileSnitch", -1)
+
+	err = ioutil.WriteFile(cfgPath, []byte(newContents), 0)
+	if err != nil {
+		return nil, err
+	}
+
+	snitchSettings, err := getSnitchSettings(opts.kubeClientset)
+	if err != nil {
+		return nil, err
+	}
+
+	glog.V(4).Info("generated snitch:")
+	glog.V(4).Info(snitchSettings)
```

Comment:
collapse into a single call to print else other log entries could be interlaced with this: `glog.V(4).Infof("generated snitch config: %v", snitchSettings)`